src/containers/Chat/ChatMessages/ChatMessage/ChatMessageType/ChatMessageType.module.css

```diff
@@ -3,7 +3,7 @@
   max-width: 280px;
   min-width: 170px;
```

Comment:
Can we also remove the minimum width since we get extra blank space if the image is of small width